Inc/ST-LIB_LOW/Sensors/DigitalSensor/DigitalSensor.hpp

```diff
@@ -0,0 +1,26 @@
+
+/*
+ * DigitalSensor.hpp
+ *
+ *  Created on: Nov 2, 2022
+ *      Author: ricardo
+ */
+
+#pragma once
+#include "ST-LIB.hpp"
+#include "ST-LIB_LOW/Sensors/Sensor/Sensor.hpp"
+
+
+class DigitalSensor : public Sensor<PinState>::Sensor{
+public:
+	DigitalSensor(Pin pin, PinState *value);
+	void exti_interruption(std::function<auto> lambda);
```

Comment:
```suggestion
	void exti_interruption(std::function<auto> action);
```
usa nombres explicativos, es una accion no? si no, ponle el nombre que tu veas que explica mejor lo que representa.